ui/src/app/views/workspace-edit/component.tsx

```diff
@@ -449,78 +532,189 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
-              <strong key = 'benefit'> expected outcome/benefit </strong>,
-              'of the research information will be posted publicly on the All of Us website to ' +
-              'inform the research participants.']} required>
-          <TextArea value={this.state.workspace.description}
-                    onChange={v => this.setState(fp.set(['workspace', 'description'], v))}/>
-        </WorkspaceEditSection>
-        <WorkspaceEditSection
-            header='Please select all data use categories that apply for your current study'
-            text='These are for informational purposes only and do not affect or configure your
-            new workspace.'>
-          <WorkspaceCategory style={{width: '100%'}}
-              item={ResearchPurposeItems.diseaseFocusedResearch}
-              value={this.state.workspace.researchPurpose.diseaseFocusedResearch}
-              onChange={v => this.updateWorkspaceCategory('diseaseFocusedResearch', v)}>
-            <TooltipTrigger
-                content='You must select disease focused research to enter a disease of focus'
-                disabled={this.state.workspace.researchPurpose.diseaseFocusedResearch}>
-              <TextInput value={this.state.workspace.researchPurpose.diseaseOfFocus}
-                         style={{
-                           width: 'calc(50% - 2rem)',
-                           border: '1px solid #9a9a9',
-                           borderRadius: '5px'
-                         }}
-                         placeholder='Name of Disease' onChange={v =>
-                  this.setState(fp.set(['workspace', 'researchPurpose', 'diseaseOfFocus'], v))}
-                         disabled={!this.state.workspace.researchPurpose.diseaseFocusedResearch}/>
-            </TooltipTrigger>
-          </WorkspaceCategory>
-          <div style={{display: 'inline-block'}}>
-            <div style={{display: 'flex'}}>
+            tooltip={toolTipText.billingAccount}/>
+        <WorkspaceEditSection header='Research Use Statement Questions'
+            text={['The AoU Research Program requires each user of AoU data to provide a \
+             meaningful description of the intended purpose of data use for each Workspace they \
+             create. The responses provided below will be posted publicly in the AoU Research Hub \
+             website to inform the AoU Research Participants.  Therefore, please provide \
+             sufficiently detailed responses at a 5th grade reading level.  Your responses will \
+             not be used to make decisions about data access.', <br/>, <br/>,
+              <i>Note that you are required to create separate Workspaces for each project
+              for which you access AoU data, hence the responses below are expected to be specific
+              to the project for which you are creating this particular Workspace.</i>
+            ]}/>
+        <WorkspaceEditSection header={researchPurposeQuestions[0].header}
```

Comment:
There's some boilerplate here that feels like it may be doable via `researchPurposeQuestions.map`. I wonder if that's usable here.

Reply:
i'm not sure if it is worth pulling out because each of the sections are so different

Reply:
^ oh nvm you meant for the checkboxes